queries/buildings.jinja2

```diff
@@ -57,10 +58,11 @@ SELECT
     NULL AS area,
     osm_id AS __id__,
     NULL AS "building:part",
+    "address" AS kind,
```

Comment:
Using `address` instead of `addr` because the extra characters aren't that precious.